sdlf-utils/pipeline-examples/datalake-workload-management/utils/cleanup.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+pflag=false
+rflag=false
+bflag=false
+
+DIRNAME=$(pwd)
+
+usage () { echo "
+    -h -- Opens up this help message
+    -p -- AWS profile
```

Comment:
-s for shared cicd account and -t for child account.

Reply:
The sdlf-wlm-integration doesnt require any deployment script changes or additions based on new changes. Standalone solution is kept as is